teacher.rb

```diff
@@ -10,3 +10,14 @@ def can_use_services?
     true
   end
 end
+
+def create_teacher(people)
+  print 'Age: '
+  age = gets.chomp
+  print 'Name: '
+  name = gets.chomp
+  print 'Specialization: '
+  specialization = gets.chomp
+  people << Teacher.new(specialization, age, name)
+  puts 'Teacher created successfully'
+end
```

Comment:
- [ ] The same thing here, Kindly check my comment above ⬆

Reply:
Done! Thanks :wink: